Automattic/wp-calypso
# Checkout: Remove old checkout PayPal payment method

#### Changes proposed in this Pull Request

This removes the PayPal payment method from old checkout which is no longer used.

#### Testing instructions

None, really. Be sure tests pass and also double-check that checkout loads without errors.
